src/vcpkg/visualstudio.cpp

```diff
@@ -337,18 +330,14 @@ namespace vcpkg::VisualStudio
                     if (fs.exists(vs_bin_dir / "amd64_arm/vcvarsamd64_arm.bat", IgnoreErrors{}))
                         supported_architectures.push_back({"amd64_arm", CPU::X64, CPU::ARM});
 
-                    if (fs.exists(vs_dumpbin_exe, IgnoreErrors{}))
-                    {
-                        const Toolset toolset = {vs_instance.root_path,
-                                                 vs_dumpbin_exe,
-                                                 vcvarsall_bat,
-                                                 {},
-                                                 major_version == "14" ? V_140 : V_120,
-                                                 major_version,
-                                                 supported_architectures};
-
-                        found_toolsets.push_back(toolset);
-                    }
+                    const Toolset toolset = {vs_instance.root_path,
```

Comment:
Since we aren't using `vs_dumpbin_exe` any more, we can remove the above lines 314-317.

Reply:
Done!